test.py

```diff
@@ -1,7 +1,6 @@
 import math
 from tabulate import tabulate
 
-
```

Comment:
Whitespace seemed irrelevant so I removed it lemme know if it's there for a reason